src/components/Buttons/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import { colors, typography } from '@magnetis/astro-tokens';
+
+import Button from '../Button';
+import { ButtonProps } from '..';
+
+const initialProps: ButtonProps = {
+  disabled: false,
+  fill: false,
+  loading: false,
+  size: 'medium',
+  text: 'text',
+  type: 'solid',
+  variant: 'primary',
+  onPress: jest.fn(),
+};
+
+describe('Button', () => {
+  it('renders correctly with default props', () => {
+    const { getByText } = render(<Button {...initialProps} />);
+
+    const button = getByText('text');
+
+    expect(button.props.style[0]).toEqual(
+      expect.objectContaining({
+        fontFamily: typography.fontFamilyBase,
+        alignItems: 'center',
+      })
+    );
+
+    expect(button.props.style[1]).toEqual(
+      expect.objectContaining({
+        fontSize: typography.fontSizeMini,
+        lineHeight: typography.fontSizeMini * 1.5,
+        color: colors.solidBrightLightest,
+      })
+    );
```

Comment:
I think can be done in one assert with `toHaveStyle`
```suggestion
    expect(button).toHaveStyle({
      fontFamily: typography.fontFamilyBase,
      alignItems: 'center',
      fontSize: typography.fontSizeMini,
      lineHeight: typography.fontSizeMini * 1.5,
      color: colors.solidBrightLightest,
    });
```

Reply:
Fixed https://github.com/magnetis/astro-native/pull/265/commits/01ad556544c0cc2de509d82ac90cf681a2306b88